Sources/protoc-gen-swiftgrpc/Generator.swift

```diff
@@ -106,18 +109,32 @@ class Generator {
     println()
     
     if options.generateClient {      
-      for service in file.services {
+      for service in file.services where options.generateImplementations {
```

Comment:
I'm not a fan of `where` clauses independent of the iterated variable; would you mind wrapping the loop in a plain old `if`? (also elsewhere, if applicable)